src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image image">
+        <img
+          class="image__iMac"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        >
+      </div>
+      <div class="card__description description">
+        <div class="description__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <p class="description__code">Код товара: 195434</p>
+        <div class="description__rate rate">
+          <span class="rate__star"></span>
+          <span class="rate__star"></span>
+          <span class="rate__star"></span>
+          <span class="rate__star"></span>
+          <span class="rate__star"></span>
+        </div>
+        <p class="description__feedback">Отзывов: 5</p>
+        <p class="description__price">Цена:</p>
+        <p class="description__amount">69 999 грн</p>
+      </div>
+      <div class="card__purchase purchase">
+        <a
+          class="purchase_btn"
```

Comment:
you have 2 BEM-blocks for 1 button: purchase and purchase_btn. Make the inner class an element, not a block
```suggestion
          class="purchase__btn"
```